pulsar-broker-common/src/main/java/org/apache/pulsar/broker/ServiceConfiguration.java

```diff
@@ -305,6 +305,13 @@
     )
     private long topicLoadTimeoutSeconds = 60;
 
+    @FieldContext(
+            category = CATEGORY_SERVER,
+            required = false,
+            doc = "defined your own decryption interface"
+    )
+    private String decryptionInterface;
```

Comment:
Users need to provide a Decryption implementation themself, But I did not find any jar load implementation. In my opinion, users need to private a location that where to find the jar? Otherwise, users need to copy jar under the classpath.

Reply:
It will not do encryotion and decryption by default, and i just make a interface that user can package their decrption implementation jar to pulsar libs dir ,then they can config the password、secret with encryted.

There also exist some sensitive config like OAuth2.0 client_secret、zookeeper SASL DIGEST-MD5 passworld 、TLS keystore password， suggest to be encryted when we use it online 

So i think maybe pulsar can provider a command encryption and decryption method in future , such as a common AES-GCM implementation